docs/theory/ensemble_based_methods.rst

```diff
@@ -318,15 +335,15 @@ where :math:`G_j = \nabla g(x_j)`.  The clever trick in ensemble methods is to r
 :math:`G_j` by an ensemble averaged sensitivity :math:`G` represented by the linear regression equation
 
 .. math::
-   C_{yx} = G C_{xx}.
+   C_{xy} = G C_{xx}.
 
-Covariances :math:`\bar{C}_{xy}`, :math:`\bar{C}_{xx}`, and :math:`\bar{C}_{dd}` are
+Covariances :math:`\bar{C}_{xy}`, :math:`\bar{C}_{yy}`, and :math:`\bar{C}_{dd}` are
 estimated from the ensemble and the state vector is updated according to:
 
 .. math::
    \begin{align}
-   x_j^a &= x_j^f + \bar{C}_{xy}(\bar{C}_{xy}^{f}\bar{C}_{xx}^{-1}\bar{C}_{xy}+\bar{C}_{dd})^{-1}(d_j-y_j^f)\\
-   X^a &= X^f + \bar{C}_{xy}(\bar{C}_{xy}^{f}\bar{C}_{xx}^{-1}\bar{C}_{xy}+\bar{C}_{dd})^{-1}(D-Y_f).
+   x_j^a &= x_j^f + \bar{C}_{xy}(\bar{C}_{yy} + \bar{C}_{dd})^{-1}(d_j - y_j^f)\\
+   X^a &= X^f + \bar{C}_{xy}(\bar{C}_{yy} + \bar{C}_{dd})^{-1}(D - Y_f).
```

Comment:
This is
1. more aligned with what we do
2. easier to read and understand

I think the previous equations perhaps is a remnant form the equations when the linear map, `H`, from forecast to response is known. Then it is natural to write it out in a similar fashion. This is however not the case for us, generally.